generator/src/main/java/com/scottlogic/deg/generator/inputs/validation/validators/GranularityConstraintValidator.java

```diff
@@ -0,0 +1,47 @@
+package com.scottlogic.deg.generator.inputs.validation.validators;
+
+import com.scottlogic.deg.generator.inputs.validation.*;
+import com.scottlogic.deg.generator.inputs.validation.messages.*;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+//TODO Consider bringing these rules to field spec restrictions
```

Comment:
Should this comment be addressed in the PR?

Reply:
refactored.